lablib/__init__.py

```diff
@@ -1,8 +1,11 @@
 from .processors import (
-    OCIOConfigFileProcessor,
     AYONHieroEffectsFileProcessor,
     AYONOCIOLookFileProcessor,
     OIIORepositionProcessor,
+)
+
+from .generators import (
+    OCIOConfigFileProcessor,
     SlateHtmlProcessor,
```

Comment:
i think we should refactor these to `OCIOConfigFileGenerator` and `SlateHtmlGenerator`